README.md

```diff
@@ -1,132 +1,127 @@
+## Installation
 
+```bash
+pip install -U pygfx glfw
+```
 
-## Introduction
+To work correctly, pygfx needs _some_ window to render to. Glfw is one
+lightweight option, but there are others, too. If you only use pygfx for
+offscreen, or notebook rendering you may choose to omit glfw. Similarly, you can
+(of course) swap it for any other wgpu-compatible canvas, e.g., PyQt, PySide, or
+wx.
 
-This is a Python render engine build on top of [WGPU](https://github.com/pygfx/wgpu-py) (instead of OpenGL).
+In addition there are some [platform
+requirements](https://github.com/pygfx/wgpu-py#platform-requirements). In
+essense, you need modern (enough) graphics drivers, and `pip >= 20.3`.
 
-We take a lot of inspiration from ThreeJS, e.g.:
+## Usage Example
 
-* Materials and Geometry are combined in world objects.
-* Decoupled cameras and controllers.
-* The code for the render engines is decoupled from the objects, allowing multiple render engines (e.g. wgpu and svg).
+> **Note**
+> A walkthrough of this example can be found in [the
+> guide](https://pygfx.readthedocs.io/en/latest/guide.html#how-to-use-pygfx).
 
-Further we aim for a few niceties:
-* Proper support for high-res screens.
-* Builtin anti-aliasing.
-* Custom post-processing steps.
-* Support for picking objects and parts within objects.
-* (approximate) order-independent transparency (OIT) (not implemented yet).
+```python
+import pygfx as gfx
 
+cube = gfx.Mesh(
+    gfx.box_geometry(200, 200, 200),
+    gfx.MeshPhongMaterial(color="#336699"),
+)
 
-## WGPU is awesome (but also very new)
+def animate():
+    rot = gfx.linalg.Quaternion().set_from_euler(
+            gfx.linalg.Euler(0, 0.01)
+        )
+    cube.rotation.multiply(rot)
 
-Working with the WGPU API feels so much nicer than OpenGL. It's well
-defined, no global state, we can use compute shaders, use storage
-buffers (random access), etc.
+if __name__ == "__main__":
+    gfx.show(cube, before_render=animate)
 
-Fair enough, the WGPU API is very new and is still changing a lot, but
-eventually it will become stable. One of the biggest downsides right
-now is the lack of software rendering. No luck trying to run wgpu on a
-VM or CI.
+```
+<img src="./docs/_static/guide_rotating_cube.gif" alt="drawing" width="400"/>
 
-Because of how Vulkan et. al. work, the WGPU API is aimed at predefining
-objects and pipelines and then executing these. Almost everything is
-"prepared". The main reasoning for this is consistency and stable drivers,
-but it also has a big advantage for us Pythoneers: the amount of code per-draw-per-object
-is very limited. This means we can have *a lot* of objects and still be fast.
 
-As an example, see `collections_line.py`: drawing 1000 line objects with 30k points each at 57 FPS (on my laptop).
+## Feature Highlights
+Some of pygfx's key features are:
 
+- SDF based rendering (example)
+- order-independent transparency (OIT) (example)
+- a shader templating system (example)
```

Comment:
The shader templating is cool, but only interesting for ppl that want to extend pygfx. Let's replace with lights etc.
```suggestion
- lights, shadows, and physically based rendering (PBR)
```